src/sidebar/components/PendingUpdatesButton.tsx

```diff
@@ -19,11 +20,18 @@ function PendingUpdatesButton({
   const store = useSidebarStore();
   const pendingUpdateCount = store.pendingUpdateCount();
   const hasPendingUpdates = store.hasPendingUpdates();
+  const applyPendingUpdates = () => streamer.applyPendingUpdates();
```

Comment:
Might be worth `useCallback`-ing this callback as otherwise `useShortcut` will remove and re-add the shortcut on each render.

Reply:
Good catch